packages/postcss-convert-values/src/lib/convert.js

```diff
@@ -31,19 +34,33 @@ function dropLeadingZero(number) {
 
   return value;
 }
-
+/**
+ * @param {number} number
+ * @param {string} originalUnit
+ * @param {Map<string, number>} conversions
+ * @return {string}
+ */
 function transform(number, originalUnit, conversions) {
   let conversionUnits = [...conversions.keys()].filter((u) => {
     return originalUnit !== u;
   });
 
-  const base = number * conversions.get(originalUnit);
+  const base = number * /** @type {number} */ (conversions.get(originalUnit));
```

Comment:
TypeScript could have caught a legitimate issue here. This would return `NaN` if the unit was unknown.